app/presenters/payment_presenter.rb

```diff
@@ -19,9 +23,28 @@ def refunded_message
     end
   end
 
+  def no_action_message
+    if already_reverted?
+      I18n.t(".reversal_forms.index.already_reversed")
```

Comment:
```suggestion
      I18n.t(".reversal_forms.index.already_reverted")
```

Nitpicking here but it'd be good to standardise on `reverted` or `reversed` 👍 Looks like `reverted` is more common